<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RedactForm</class>
 <widget class="QWidget" name="RedactForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>200</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Редактирование заметки</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="NameLabel">
       <property name="text">
        <string>Имя заметки</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="NameValueLabel">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="ChangeNameLabel">
       <property name="text">
        <string>Изменить на:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ChangeNameEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="DateLabel">
       <property name="text">
        <string>Дата срока</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="DateValueLabel">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="ChangeDateLabel">
       <property name="text">
        <string>Изменить на:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="dateTimeEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="ApplyBtn">
     <property name="text">
      <string>Применить</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
